Author: Chris Kuehl <ckuehl@ocf.berkeley.edu>
Subject: Unvendor rbenv
Forwarded: not-needed
Description: Upstream includes rbenv tarballs into their source which are
 distributed with the package so that users needn't install rbenv manually.
 .
 We just add `Depends: nodeenv` and strip the tarballs to comply with Policy
 (``4.13 Convenience copies of code'').

--- a/pre_commit/languages/ruby.py
+++ b/pre_commit/languages/ruby.py
@@ -3,6 +3,7 @@
 
 import contextlib
 import io
+import os
 import shutil
 
 from pre_commit.languages import helpers
@@ -30,22 +30,7 @@
 
 def _install_rbenv(repo_cmd_runner, version='default'):
     directory = helpers.environment_dir(ENVIRONMENT_DIR, version)
-
-    with tarfile_open(resource_filename('rbenv.tar.gz')) as tf:
-        tf.extractall(repo_cmd_runner.path('.'))
-    shutil.move(
-        repo_cmd_runner.path('rbenv'), repo_cmd_runner.path(directory),
-    )
-
-    # Only install ruby-build if the version is specified
-    if version != 'default':
-        # ruby-download
-        with tarfile_open(resource_filename('ruby-download.tar.gz')) as tf:
-            tf.extractall(repo_cmd_runner.path(directory, 'plugins'))
-
-        # ruby-build
-        with tarfile_open(resource_filename('ruby-build.tar.gz')) as tf:
-            tf.extractall(repo_cmd_runner.path(directory, 'plugins'))
+    os.makedirs(os.path.join(repo_cmd_runner.path(directory), 'bin'))
 
     activate_path = repo_cmd_runner.path(directory, 'bin', 'activate')
     with io.open(activate_path, 'w') as activate_file:
--- a/setup.py
+++ b/setup.py
@@ -33,9 +33,6 @@
         'pre_commit': [
             'resources/hook-tmpl',
             'resources/pre-push-tmpl',
-            'resources/rbenv.tar.gz',
-            'resources/ruby-build.tar.gz',
-            'resources/ruby-download.tar.gz',
         ]
     },
     install_requires=[
